### ISSUE TITLE
Steps count in dashboard displays 0 even though there are logged steps

### ISSUE THREAD

**Issue by performrumpf** [extent-framework/extentreports-java#18]
In version 4.0.6 the dashboard displays number of tests and test steps. The number of steps is always 0 no matter how many steps you log.

![dashboard_view_4 0 6](https://user-images.githubusercontent.com/39511399/50293821-bfacc880-0474-11e9-8457-efd29880faa3.png)

**Comment by anshooarora** [comment]
This is due to the rewrite of `SessionStatusStats` class to only include Tests/Children(steps) as the stats mechanism.  Log/events are no longer considered which is why in your case there is just 1 graph showing.  There is currently an enhancement open for our professional version, and once the best way to enable this is known, it will come to the community version also.

**Comment by performrumpf** [comment]
Ah, so it was a feature for community version.

I changed the framework so that for every "test step" he will create a new node and log every sub test step into that node.

(condition: parent != null && parent.parent == null)

**Comment by formynet** [comment]
Do you know when this issue is planned to be fixed?

**Comment by ohlori** [comment]
Still existing... :(

**Comment by SubrotoMohanta** [comment]
Use createnode("details") This will log the steps

**Comment by anshooarora** [comment]
This feature will be available version `4.10.0` onwards.  It is currently being worked on and will share the SNAPSHOT version as it becomes ready.

**Comment by anshooarora** [comment]
This feature is available in `4.10.0-SNAPSHOT`:

```
<groupId>com.aventstack</groupId>
<artifactId>extentreports</artifactId>
<version>4.1.0-SNAPSHOT</version>
```

**Comment by melleck** [comment]
still existing in 4.1.0
